prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -391,29 +384,28 @@ In most cases, this is probably a bad idea.
 
 2. The request payload can store a `buyeruid` for each Bidder by defining `request.user.ext.prebid.buyeruids` like so:
 
-```
-{
-  "user": {
-    "ext": {
-      "prebid": {
-        "buyeruids": {
-          "appnexus": "some-appnexus-id",
-          "rubicon": "some-rubicon-id"
+    ```
+    {
+      "user": {
+        "ext": {
+          "prebid": {
+            "buyeruids": {
+              "bidderA": "some-biddera-id",
+              "bidderB": "some-bidderb-id"
+            }
+          }
         }
       }
     }
-  }
-}
-```
+    ```
```

Comment:
Continuing to break away from always using appnexus and rubicon in examples. Let's be more inclusive. :) Also fixed indentation such that bullet point 3 is not changed to 1 by markdown.